Enable fpm to create a new package (#102)
fortran-lang/fpm: README.md

On Linux, the above command installs `fpm` to `${HOME}/.local/bin`.

### Creating a new project

> Creating a new fpm project is as simple as running the command `fpm new project_name`.

Reviewer: ```suggestion
Creating a new fpm project is as simple as running the command `fpm new <project-name>`.
```

This will create a new folder in your current directory with the following contents
and initialized as a git repository.

> * `fpm.toml` with your project's name and some default standard meta-data

Reviewer: ```suggestion
* `fpm.toml` with your project's name and some default standard metadata
```

* `README.md` with your project's name
* `.gitgnore`

> * `src/project_name.f90` with a simple hello world subroutine

Reviewer: ```suggestion
* `src/<project-name>.f90` with a module containing a simple hello world subroutine
```

* `app/main.f90` (if `--with-executable` flag used) a program that calls the subroutine

> * `test/main.f90` (if `--with-test` flag used) an empty test program

Reviewer: Could be tackled in a separate PR, but I suggest we use `--bin` instead of `--with-executable` (and also make it the default). Also, add an empty test program by default, as this will encourage all new projects to have tests.
